### ISSUE TITLE
Dark Mode Customization

### ISSUE THREAD

**Issue by Oronhyatekha** [blyad2137/google-dark-theme#28]
Would be nice, to choose between a "dark blue" and a "dark grey" color set. So it fits the dark mode of the browser window.

**Comment by blyad2137** [comment]
[This UserStyle was never meant to be customizable, `dark-blue` and `dark-grey` are used in different places to suit their environment. To achieve such thing I would have to replace ~2049 lines of code, you could think that I can replace them using program such as Notepad++, well yes of course, but there is still a factor how it will look. 
I would also have to check if the contrast of colors match their surrounding(if they don't stand out too much or so), this means I would need to check those 2049 lines by hand and see how they look.](https://www.youtube.com/watch?v=RxnNjK2dPXI)

tl;dr Maybe in another life.
